src/app/components/TopicForm/TopicForm.tsx

```diff
@@ -41,36 +42,27 @@ export default function TopicForm({
 
   return (
     <TopicFormContainer onSubmit={handleSubmit}>
-      <InputWrapper>
-        <FormLabel htmlFor="title">Title*</FormLabel>
-        <FormInput
-          type="text"
-          name="title"
-          required
-          value={title}
-          onChange={event =>
-            event.target.value.length <= MAX_TITLE_LENGTH &&
-            setTitle(event.target.value)
-          }
-        />
-        <TextLimitDisplay diff={titleDiff}>{`${titleDiff}`}</TextLimitDisplay>
-      </InputWrapper>
-      <InputWrapper>
-        <FormLabel htmlFor="description">Description</FormLabel>
-        <FormTextArea
-          name="description"
-          rows={10}
-          value={description}
-          onChange={event =>
-            event.target.value.length <= MAX_DESCRIPTION_LENGTH &&
-            setDescription(event.target.value)
-          }
-        />
-        <TextLimitDisplay diff={descriptionDiff}>
-          {`${descriptionDiff}`}
-        </TextLimitDisplay>
-      </InputWrapper>
-
+      <FormInput
+        type="text"
+        name="title"
+        required
+        diff={titleDiff}
+        value={title}
+        onChange={event =>
+          event.target.value.length <= MAX_TITLE_LENGTH &&
```

Comment:
What the hell are these => <= doing? o.O

Reply:
I think it's the "kleiner gleich" sign, but I was also a little confused at first 😅